dotcom-rendering/scripts/env/check-deps.js

```diff
@@ -0,0 +1,9 @@
+const pkg = require('../../package.json');
+
+if (pkg.devDependencies) {
+	const { warn, log } = require('./log');
+
+	warn('Don’t use devDependencies');
+	log('See https://github.com/guardian/dotcom-rendering/pull/4001');
+	process.exit(1);
+}
```

Comment:
Could we check all `package.json` files?

Reply:
Maybe but I didn't feel confident.